src/Helpers/Builder/InboundRouteParams.php

```diff
@@ -0,0 +1,140 @@
+<?php
+
+namespace MailerSend\Helpers\Builder;
+
+use Tightenco\Collect\Contracts\Support\Arrayable;
+
+class InboundRouteParams implements Arrayable, \JsonSerializable
+{
+    protected string $domainId;
+    protected string $name;
+    protected bool $domainEnabled;
+    protected ?string $inboundDomain = null;
+    protected $catchFilter = null;
+    protected $matchFilter = null;
```

Comment:
The API docs have the specification of what is required, and since it's going to return a validation error the users should adjust. Just so we don't validate everything in the SDK. But we could validate somethings as well... Wdyt @fosron?